kubernetes/README.md

```diff
@@ -0,0 +1,41 @@
+# Kubernetes CoreDNS Update Release Process
+
+This document helps define the steps to be followed after a CoreDNS release to update the image/manifests for CoreDNS in the Kubernetes (and related) projects.
+After a [CoreDNS release](https://github.com/coredns/release) has been completed by one of the maintainers:
+
+
+1. Update the [CoreDNS deployment](https://github.com/coredns/deployment/blob/master/kubernetes/coredns.yaml.sed) for the latest release version and ensure all the CI tests pass. But do not merge yet.
+    * Open a PR in the [Kubernetes main repository](https://github.com/kubernetes/kubernetes) that tests the new image (in Kubernetes e2e) before it is pushed to gcr.io. 
+      Clearly title the issue that this is a test PR, not intended for merge. *Do not merge*.
+    * If the Kubernetes pre-submit e2e tests pass, then close the above kubernetes/kubernetes PR, and merge the coredns/deployment PR.
+
+2. Open an issue in the [Kubernetes main repository](https://github.com/kubernetes/kubernetes) to get the CoreDNS image pushed to the gcr.io repository. 
+Example issue opened for previous releases: https://github.com/kubernetes/kubernetes/issues/69983
+
```

Comment:
I guess we should say here that we need to assign to a Google person that has permission to push to gcr.io. By default, it is Tim Hockin, SIG-Network.